src/main/webapp/javascript/userSignupSignin.js

```diff
@@ -101,6 +101,7 @@ async function getUsername(username) {
     if(msgFromResponse.match(numbers)){
         msgDiv.innerHTML = `<p class="success">Success!</p>`;
 
+        sessionStorage.setItem("userName", username);
         sessionStorage.setItem("userId", msgFromResponse);
```

Comment:
Not too sure why we need to setItem for username and userid again here at login? Aren't they already in storage?

Reply:
this is for user login and their browser may not have the cache from signing up. In this case, userid is fetched from backend